prow/plugins/updateconfig/updateconfig.go

```diff
@@ -62,28 +62,33 @@ type githubClient interface {
 }
 
 type kubeClient interface {
+	GetNamespace() string
 	ReplaceConfigMap(name string, config kube.ConfigMap) (kube.ConfigMap, error)
 }
 
 func handlePullRequest(pc plugins.PluginClient, pre github.PullRequestEvent) error {
 	return handle(pc.GitHubClient, pc.KubeClient, pc.Logger, pre, maps(pc))
 }
 
-func maps(pc plugins.PluginClient) map[string]string {
+func maps(pc plugins.PluginClient) map[string]plugins.ConfigUpdate {
 	return pc.PluginConfig.ConfigUpdater.Maps
 }
 
-func update(gc githubClient, kc kubeClient, org, repo, commit, filename, name string) error {
+func update(gc githubClient, kc kubeClient, org, repo, commit, filename, name, namespace string) error {
 	content, err := gc.GetFile(org, repo, filename, commit)
 	if err != nil {
 		return fmt.Errorf("get file err: %v", err)
 	}
 
 	c := string(content)
+	if namespace == "" {
+		namespace = kc.GetNamespace()
+	}
```

Comment:
This and the `GetNamespace` function are unnecessary. If `cm.Namespace == ""` when `ReplaceConfigMap` is called, the kube client's default namespace is used.

Reply:
done